documentation/modules/migration-plan-options-ui.adoc

```diff
@@ -8,30 +8,41 @@
 
 On the *Plans for virtualization* page of the {ocp} web console, you can click the {kebab} beside a migration plan to access the following options:
 
-* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs* changes to *Download logs* and then click the button to download the logs.
-* *Edit*: Edit the details of a migration plan. You cannot edit a migration plan while it is running or after it has completed successfully.
-* *Duplicate*: Create a new migration plan with the same virtual machines (VMs), parameters, mappings, and hooks as an existing plan. You can use this feature for the following tasks:
+* *Edit Plan*: Edit the details of a migration plan. If the plan is running, or it has been completed successfully, the following options cannot be edited:
+
+** All properties on the *Settings* section of the *Plan details* page. For example, warm or cold migration, target namespace, and preserve static IPs.
+** The plan's mapping on the *Mappings* tab.
+** The hooks listed on the *Hooks* tab.
+
+* *Start migration*: Active only if relevant.
+* *Restart migration*: Restart a migration that was interrupted. Before choosing this option, make sure there are no error messages. If there are, you need to edit the plan.
+* *Cutover*: Warm migrations only. Active only if relevant. Clicking *Cutover* opens the the *Cutover* window, which supports the following options:
+
+** *Set cutover* Set the date and time for a cutover.
+** *Remove cutover* Remove a planned cutover.
+
+* *Cancel scheduled cutover*: Cancel a scheduled cutover migration for a warm migration plan. Active only if relevant.
```

Comment:
The `Remove cutover` and `Cancel scheduled cutover` options are actually the same.
When clicking on the `Remove cutover` action, you are actually canceling it. 
there is no `Cancel scheduled cutover` button. I would change it to:

 *Remove cutover*: Cancel a scheduled cutover migration for a warm migration plan. Active only if relevant.

Reply:
Fixed -- shortened the description since the context is already a warm migration.